src/metabase/models/card.clj

```diff
@@ -36,6 +36,7 @@
   :m/Card)
 
 (methodical/defmethod t2/table-name :m/Card [_model] :report_card)
+(derive :m/Card :metabase/models)
```

Comment:
@camsaul wdyt if we make a macro wrapper in ` metabase.models.interface` that does both define table-name and derive this for base models?